src/lib/Scenes/Map/__tests__/GlobalMap-tests.tsx

```diff
@@ -1,24 +1,32 @@
 import { CityFixture } from "lib/__fixtures__/CityFixture"
 import { renderRelayTree } from "lib/tests/renderRelayTree"
 import { graphql } from "react-relay"
-import { GlobalMap } from "../GlobalMap"
+import { GlobalMapContainer } from "../GlobalMap"
 
 jest.unmock("react-relay")
 jest.mock("@mapbox/react-native-mapbox-gl", () => ({
   StyleSheet: {
     create: jest.fn(),
+    identity: jest.fn(),
+    source: jest.fn(),
   },
   InterpolationMode: {
     Exponential: jest.fn(),
   },
   MapView: jest.fn(),
   setAccessToken: jest.fn(),
+  UserTrackingModes: jest.fn(),
 }))
 
 describe("GlobalMap", () => {
-  xit("renders correctly", async () => {
+  it("renders correctly", async () => {
     await renderRelayTree({
-      Component: GlobalMap,
+      componentProps: {
```

Comment:
We'll want to mock the `viewer` object in this `mockData`. Generally, I think the mockData object should match the top level of the corresponding query. So ```mockData: {
        viewer: viewerFixture,
      }```

Reply:
Not sure I follow what you mean. Would you please elaborate? The `mockData` shape maps this query mockData > viewer > city > cityFixture. 

The props passed to `componentProps` don't go in the mockData object as @alloy pointed out in a previous code review so they are being passed here. The `safeAreaInsets` prop comes Mapbox (not the Graphql data) so it has to be mocked in the `componentProps` field.

Reply:
Ah yes you're right! Usually viewer is used when wrapping two or more queries so I thought that was the case here.